src/providers/SignatureProvider.tsx

```diff
@@ -15,6 +16,7 @@ const getDefaultInputElementOptions = (type: InputElementOptions["variant"]): In
     isBold: false,
     isItalics: false,
     style: { fontWeight: "normal", fontStyle: "normal" },
+    color: "",
```

Comment:
Lets make the default color `black`

```suggestion
    color: parseColor("#000000"),
```

```javascript
import { parseColor } from "@react-stately/color";
```